docs/UserGuide.md

```diff
@@ -381,17 +381,38 @@ Edits a specific interview in the list of interviews.
 
 <u>Format:</u>
 
-    edit_i <INDEX> [position=POSITION]... [c=<CANDIDATE INDEX>]... [date=DATE]... [time=TIME]...
-    [duration=DURATION]... [interviewed=STATUS]...
+    edit_i <INDEX> [position=POSITION]... [date=DATE]... [time=TIME]... [duration=DURATION]... [interviewed=STATUS]...
 
 <u>Example:</u>
-`edit_i 2 c=1 2 date=18/10/2021 time=1400`
-* Edits the second interview in the interview list and updates the candidate set, date and time of the interview.
+`edit_i 2 date=18/10/2021 time=1400`
+* Edits the second interview in the interview list and updates the date and time of the interview.
+  
 * All input fields should be provided in the correct format. Please refer to notes on interview command format shown
   above to see what constitutes a valid input.
-* At least one input field must be edited. For instance, in the above example, three input fields have been edited - 
-  `c=1 2` for the candidates assigned to the interview, `date=18/10/2021` for the date of the interview and `time=1400`
-  for the time the interview is scheduled for.
+  
+* At least one input field must be edited. For instance, in the above example, two input fields have been edited -
+  `date=18/10/2021` for the date of the interview and `time=1400` for the time the interview is scheduled for.
+
+<div markdown="block" class="alert alert-info">
+
+**NOTE:**
+`edit_i` cannot be used to edit the candidates assigned to the specified interview.
+  Please refer to `assign` and `unassign` commands below to see how you can add and remove candidates
+  from a scheduled interview.
+
+</div>
+
+  <br>
+  <br>
+
+#### <u>List all interviews:</u> `list_i`
+
+*Displays a list of all the interviews stored in the application.*
+
+<u>Format:</u>
+
+    list_i
+
```

Comment:
Didn't your previous UG update put list before delete?

Reply:
there are no conflicts with the previous ug update even after pulling